Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,78 @@
+
+Select Canvas - For each new session, the "Drawing Screen" would force the user to select a canvas. Canvas selection could be a pop-up, and would allow the creation of a blank/new canvas or the selection of one that's
+already been worked on (loading feature).
+
+Draw - Drawing would happen with a default brush given to the user. Holding down on the canvas would paint with the selected brush and color.
+
+Tap Drawing Options Menu - By tapping this button, a few additional "bubbles" pop-up on the screen. These are additional options such as, selecting another canvas, undoing changes, redoing changes, sharing on
+social media, or saving to device.
+
+Select Another Canvas - Pop-up from the "Drawing Options Menu", facilitates canvas selection. Users can pick another one of their saved canvases or create a blank one for a new artwork.
+
+Undo - Another pop-up from the "Drawing Options Menu", facilitates undoing your previous changes. Can support up to X number of previous changes.
+
+Redo - Another pop-up from the "Drawing Options Menu", facilitates redoing your previous changes. Again, can support up to X number of changes.
+
+Select From Right-hand Side Menu - Apart from the "Drawing Options Menu", users can interact with additional buttons on the right-hand side. These would hold features such as: color selection, brush selection and
+modification, eraser selection and modification, open color palette, save canvas.
+
+Select Color - Tapping the "Select Color" button would open a color picker where users could define the details of the colur they would like to use. This color picker could have another feature "Add color to palette"
+which would allow them to save colors to their selected palette.
+
+Change Color - Tapping the "Select Color" button again, would open the color picker again, and would allow users to alter the color they have selected. (Could be the same use case to be honest).
+
+Add Color to Palette - Tapping the "Select Color" button again, would allow users to add a selected color to any of the color palettes they have or to a new 'unsaved palette' -> this would be their way of creating a new
+palette.
+
+Select Brush - Tapping the "Select Brush" button, would open a brush selector. It could have 'templated brushes', like: smooth brush, hard brush, thick brush, thin brush, but would also host a small interface so
+users can alter brush behaviour.
+
+Change Brush Behaviour - Tapping the "Select Brush" button, would open the brush selector again, but the small interface would allow users to alter their brush behaviour.
+(This could be an overlap with the Select Brush use case).
+
+Select Color from Palette - Tapping the "Color Palettes" button, would open an interface hosting saved palettes and the collection of colors inside them. Users could pick a palette they have previously saved and/or
```

Comment:
I think it will probably be just a single color palette and one slot for your own selection